### DISCUSSION TITLE
Accessing Raw SHAP Values

### DISCUSSION THREAD

**Discussion by EvanELC** [shap/shap#2896] [Q&A]
Hello,

I am conducting some research in explainable A.I. and am using SHAP as the primary metric for showing the explainability of a model. I am in the process of coming up with ways to compare the SHAP values between models. To do this I am trying to access the raw values that are returned from the DeepExplainer's shap_values() function.

From my understanding, I thought SHAP values are supposed to be between -1 and 1, but when I print out the returned list of values they can get as high as (-)10.

How can I access the true SHAP values, or is the list from shap_values() already these values and there is something else happening to make them from -1 to 1?

Thanks for the help!

**Comment by thatlittleboy** [comment]
The discussion in this [issue](https://github.com/slundberg/shap/issues/350) might be of interest to you @EvanELC
